style.css

```diff
@@ -1,3 +1,230 @@
+/* GENERIC STYLING */ 
+
 * {
 	box-sizing: border-box;
+	font-family: Helvetica;
+	font-size: 10px;
+}
+
+h1 {
+	font-family: cursive;
+	color: #ebedf0; /*a grey-ish white*/
+	font-size: 7rem;
+	margin: 20px auto;
+}
+
+h2 {
+	font-size: 3rem ;
+	color: #d65a64; /* cute pink colour*/	
+	margin: 0 50px;
+}
+
+h3 {
+	color: #ebedf0;	
+	font-size: 2rem;	
+}
+
+p {
+	color: #ebedf0;
+	font-size: 1.8rem;
+}
+
+a {
+	text-decoration: none;
+	font-weight: bold;
+	color: #d65a64; 
+	font-size: 1.8rem;
+}
+
+.aboutMe_section, .portfolio_section, .contact_section {
+	padding-top: 30px;
+	padding-bottom: 30px;
+}
+
+/* HEADER SECTION*/
+
+header {
+	display: flex;
+	flex-direction: column;
+	background-color: #262323;
+	padding: 20px;	
+}
+
+nav {
+	display: flex;
+	margin: 20px auto;
+	gap: 20px;
+	justify-content: space-evenly;
+}
+
+nav a {
+	font-size: 2.5rem;
+}
+
+
+/* INTRODUCTION/WELCOME SECTION */
+
+.introduction_section {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-end;
+	justify-content: center;
+	column-gap: 20px;
+	width: 100%;
+	height: 900px;
+	background-image: url(images/horse_head_nebula.jpg); /* a place-holder image */ 
+	background-size:cover; /* allows to get the background in the whole section*/
+}
+
+.introduction_section h2 {
+	color: white;
+	font-size: 5rem;
+	font-variant: small-caps;
+	margin-left: 30%;
+	margin-right: 15%;
+}
+
+.introduction_section p {
+	background-color: rgba(0,0,0,0.5);
+	color: white;
+	margin-left: 30%;
+	margin-right: 15%;	
+	font-size: 3.5rem;
+	font-variant: small-caps;	
+}
+
+
+/* ABOUT ME SECTION */
+
+.aboutMe_section {
+	display: flex;
+  flex-direction: column;
+	align-items: flex-start;
+	justify-content: center;
```

Comment:
as far as i can tell, these 4 lines of flex code don't make a difference, if i switch them off it appears to look and behave the same